src/application-insights/azext_applicationinsights/_help.py

```diff
@@ -88,6 +88,21 @@
           az monitor app-insights component delete --app demoApp -g demoRg
 """
 
+helps['monitor app-insights component billing'] = """
+    type: group
+    short-summary: Manage an Application Insights component billing features.
+"""
+
+helps['monitor app-insights component billing show'] = """
+    type: command
+    short-summary: Show the billing features of an Application Insights resource.
```

Comment:
No example?

Reply:
I can add one. But it's quite simple.

Reply:
simple but may be useful to users.

---
In reply to: [383775971](https://github.com/Azure/azure-cli-extensions/pull/1316#discussion_r383775971) [](ancestors = 383775971)